README.md

```diff
@@ -1,5 +1,5 @@
-<!-- ![](https://img.shields.io/badge/Microverse-blueviolet) -->
-![This is an image](./images/check.jpg)
+
+![This is an image]()
```

Comment:
- [ ] This image is still missing, I strongly suggest updating it, the README file is very important for people to know what your project is about.

Reply:
fixed!